Tasks/DockerV2/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 156,
-        "Patch": 2
+        "Minor": 157,
```

Comment:
158?

Reply:
Will be porting to M157. As there is no change, I wanted to keep the same in master.